### ISSUE TITLE
Errors in PHPUnit tests on PHP 7.1

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpointsorg#56]
Just happened to notice that on PHP nightly we are getting [errors in our PHPUnit tests](https://travis-ci.org/WordPoints/wordpointsorg/jobs/153032705#L1104):

```
1) WordPointsOrg_Iframe_Module_Changelog_Test::test_displays_changelog

Parameter 1 to wp_default_scripts() expected to be a reference, value given

/tmp/wordpress/src/wp-includes/plugin.php:600
/tmp/wordpress/src/wp-includes/class.wp-scripts.php:162
/tmp/wordpress/src/wp-includes/class.wp-scripts.php:144
/home/travis/build/WordPoints/wordpointsorg/tests/phpunit/tests/iframe-module-changelog.php:58
```

The issue is really upstream in WordPress, I believe, but we'd need to investigate further to make sure. I think that most likely [this](https://wiki.php.net/rfc/this_var#disable_ability_to_re-assign_this_indirectly_through_reference) is the cause.

**Comment by JDGrimes** [comment]
WordPress [has now noticed this as well](https://core.trac.wordpress.org/ticket/37625#comment:1), but [probably won't pursue a fix until 7.1 is more stable](https://core.trac.wordpress.org/ticket/37625#comment:2).

**Comment by JDGrimes** [comment]
Issues seem to be fixed in WordPress 4.7.
